src/main/java/seedu/duke/command/AddToWatchlistCommand.java

```diff
@@ -1,17 +1,29 @@
 package seedu.duke.command;
 
 import seedu.duke.anime.AnimeData;
-import seedu.duke.bookmark.Bookmark;
 import seedu.duke.exception.AniException;
 import seedu.duke.human.UserManagement;
+import seedu.duke.storage.Storage;
 import seedu.duke.watchlist.Watchlist;
 
 import java.util.ArrayList;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 public class AddToWatchlistCommand extends Command {
+    private static final String ADD_OPTION = "-a";
+    
+    private String option;
+    private String animeName = "";
+    private static Logger logger = Logger.getLogger("AddToWatchlistLogger");
```

Comment:
Perhaps we could standardize our logger to be something along the lines of private static `Logger LOGGER = Logger.getLogger("<CLASS>.class.getName()");`

Reply:
The logger has been updated for all the classes that I could find it in. Thanks for pointing it out!